gnucash/gtkbuilder/window-autoclear.glade

```diff
@@ -71,18 +68,59 @@
               <object class="GtkLabel" id="label1">
                 <property name="visible">True</property>
                 <property name="can_focus">False</property>
-                <property name="label" translatable="yes">&lt;b&gt;Auto-Clear Information&lt;/b&gt;
-Use this dialog if you want GnuCash to automatically find which transactions are cleared, given an ending balance. For example, said ending balance can be the current balance given by your online bank.
-
-&lt;b&gt;Note&lt;/b&gt;: The complexity of this algorithm is quite high (&lt;a href="https://en.wikipedia.org/wiki/Knapsack_problem#Computational_complexity"&gt;O(nW)&lt;/a&gt;). Use this only on a dozen of uncleared splits.</property>
-                <property name="use_markup">True</property>
+                <property name="label" translatable="yes">About Auto-Clear</property>
+                <style>
+                  <class name="gnc-class-highlight"/>
```

Comment:
Looking at this again, I think I would swap these, make the first label style 'gnc-class-emphasize-label' and add both to the second 'gnc-class-emphasize-label' and 'gnc-class-highlight'

I suppose one could go further and create 'gnc-primary-title', 'gnc-secondary-title' which could start of the same as 'gnc-class-emphasize-label' and also 'gnc-class-bold' just for bold text.

Reply:
As I try to have a semantic view, obvisious the first is a title, but the second markup is more an admonition (note, info, tip, important, warning, caution, danger) than a subtitle. They can also be complemeted or replaced by icons.

Where is this stylesheet else used?

Usually (HTML, Docbook) "emphasis" means italic and "strong" is used for bold. Should we not lean to that standard?

Reply:
> Where is this stylesheet else used?

The class 'gnc-class-emphasize-label', the name was what I came up with at the time, is used in a dialog that is shown when you change the account type. This class name can easily be changed to what ever, maybe ...
.gnc-class-underline-title {
  font-size: large;
  font-weight: bold;
  text-decoration-line: underline;
}
.gnc-class-title {
  font-size: large;
  font-weight: bold;
}
.gnc-class-strong {
  font-weight: bold;
}